dsl_parser/utils.py

```diff
@@ -317,3 +318,16 @@ def get_class(class_path):
                          .format(class_module_str, class_name))
 
     return getattr(module, class_name)
+
+
+def generate_namespaced_value(namespace, value):
+    return "{0}{1}{2}".format(namespace, NAMESPACE_DELIMITER, value)
+
+
+def find_value_namespace(value):
+    return value.find(NAMESPACE_DELIMITER)
+
+
+def remove_value_namespace(value):
+    value_namespace_format = '.*{0}'.format(NAMESPACE_DELIMITER)
+    return re.sub(value_namespace_format, '', value)
```

Comment:
Couldn't this just be done with a `replace`?

Reply:
I don't know what is the namespace value at that point